packages/api/cms-api/src/user-permissions/content-scope.service.ts

```diff
@@ -105,6 +105,9 @@ export class ContentScopeService {
             return GqlExecutionContext.create(context).getArgs();
         } else {
             const request = context.switchToHttp().getRequest();
+            if (request.query?.scope) {
```

Comment:
```suggestion
            if (typeof request.query?.scope === "string") {
```

This would be more fitting for the PR title